docs/IntegrationWithExistingApps.md

```diff
@@ -599,10 +605,10 @@ This is only really used in dev mode when reloading JavaScript from the developm
 
 ## Add native code
 
-You need to add some native code in order to start the React Native runtime and get it to render something. To do this, we're going to create an `Activity` that creates a `ReactRootView`, starts a React application inside it and sets it as the main content view.
+You need to add some native code in order to start the React Native runtime and get it to render something. To do this, we're going to create an `AppCompatActivity` that creates a `ReactRootView`, starts a React application inside it and sets it as the main content view.
```

Comment:
You always want to use `AppCompatActivity` if you are targetting Android version <5. `AppCompatActivity` mainly adds support for `ActionBar` for older OS versions. The appcompat package from Google also enables you to use Material Design on Android <5. So you always want to use `AppCompatActivity` when targetting Android < 5. I also have the impression that using `Activity` instead of `AppCompatActivity` caused some crashes, but I'm not sure. Maybe we can change this into a note?